components/Card/index.js

```diff
@@ -66,9 +91,26 @@ export default function Card({ initialItems, items }) {
           })}
         </ul>
       )}
+
       <ul style={{ listStyle: "none" }}>
-        {selectedItems.map((selectedItem) => {
-          return <li key={selectedItem.name}>{selectedItem.name} </li>;
+        {selectedItems.map((selectedItem, index) => {
+          return (
+            <li key={selectedItem.name}>
+              {" "}
+              <button
+                type="button"
+                onClick={() => {
+                  const newSelectedItems = selectedItems.filter(
+                    (item, i) => i !== index
```

Comment:
please don't use abbreviations like i, you can use currentIndex for example.

Reply:
but it would be better to use the unique identifier you used for the key as well: item.name !== selectedItem.name

Reply:
i changed it